docs/guides/server/db.adoc

```diff
@@ -128,20 +128,27 @@ Then continue configuring the database as described in the next section.
 
 For each supported database, the server provides some opinionated defaults to simplify database configuration. You complete the configuration by providing some key settings such as the database host and credentials.
 
-. Start the server and set the basic options to configure a database
+The configuration can be set during a *build command* OR a *start command*:
```

Comment:
Should it be mentioned that other non-optimized commands (import, export, bootstrap-admin) can configure the database as well?

Reply:
I'll mention import and export, but bootstrap-admin seems unknown to kc.sh.

Reply:
It's new to version 26